rfc-029-source-code-should-be-open-source-by-default.md

```diff
@@ -0,0 +1,144 @@
+# Source code should be open source by default
+
+## Summary
+
+We believe in working in the open and sharing our work with others. We
+think that nothing should be secret unless there’s a good reason for
```

Comment:
One very minor and non-blocking thing - in other RFCs we use straight quotes rather than curly.

Reply:
Is that a conscious decision or is it just that people usually write in text editors that don't do smart quotes?

Reply:
I don't think it's a conscious decision, I have no preference either way and am happy to mix, just spotted it and went "that looks unusual".

Reply:
👍